internal/cmd/envs.go

```diff
@@ -125,3 +134,78 @@ coder envs --user charlie@coder.com ls -o json \
 		},
 	}
 }
+
+func createEnvCommand() *cobra.Command {
+	var (
+		org    string
+		img    string
+		tag    string
+		follow bool
+	)
+
+	cmd := &cobra.Command{
+		Use:    "create [environment_name]",
+		Short:  "create a new environment.",
+		Args:   cobra.ExactArgs(1),
+		Hidden: true,
+		Long:   "Create a new environment under the active user.",
+		Example: `# create a new environment using default resource amounts
+coder envs create --image id-of-imported-image my-env-name
```

Comment:
The example should be "runnable" in theory for some data. Let's use a fake image id here.